### DISCUSSION TITLE
[rejected] Możliwość ponownego złożenia wniosku o zwrot prowizji

### DISCUSSION THREAD

**Discussion by pawelosajda** [allegro/allegro-api#5889] [Suggestions - rejected]
Dzień dobry,

Opiszę pewien przypadek

1. Klient licytuje ofertę, nie opłaca jej
2. Zostaje złożony wniosek o zwrot prowizji i zostaje przyznany
3. Potem klient decyduje się opłacić zamówienie, zamówienie jest wysłane 
4. Klient zwraca przedmiot
5. Dlatego znów wysyłany jest wniosek o zwrot prowizji, jednak drugi raz nie można wystąpić o zwrot prowizji dla pozycji o takim samym line id

Czy znany jest Państwu taki scenariusz i czy planowane jest rozwiązanie które umożliwi złożenie dla jednej pozycji zamówienia kilka wniosków o zwrot prowizji?

**Comment by MaciejFrackowiak** [comment]
To jest prawidłowe działanie mechanizmu, oddające flow dostępny również na stronie WWW - w takich przypadkach niezbędny jest kontakt poprzez formularz kontaktowy. Opis procesu znajdziesz w [Pomocy Allegro](https://allegro.pl/pomoc/dla-sprzedajacych/rabat-transakcyjny-zwrot-prowizji/czy-moge-odzyskac-prowizje-od-sprzedazy-jesli-umowa-nie-zostala-zrealizowana-EY4g8OgmbtX).
